SPECS/xapi.spec

```diff
@@ -477,6 +480,8 @@ Coverage files from unit tests
 - Enforce update of xapi.conf when it's updated in the RPM
 - Add warning on top of xapi.conf to prevent user modification
 - Add additional experimental sm drivers to xapi.conf
+- /etc/xapi.conf.d/allow-sched-gran.conf becomes /etc/xapi.conf.d/00-XCP-ng-allow-sched-gran.conf
```

Comment:
1.249.3-1.3 has already been built => release bump and new changelog entry

When in doubt, check https://koji.xcp-ng.org/packageinfo?packageID=387